crates/database/src/auction_prices.rs

```diff
@@ -45,6 +45,15 @@ pub async fn fetch(
     Ok(prices)
 }
 
+pub async fn fetch_latest_prices(ex: &mut PgConnection) -> Result<Vec<AuctionPrice>, sqlx::Error> {
+    const QUERY: &str = "SELECT * FROM auction_prices WHERE auction_id = (
+        SELECT MAX(auction_id)
+        FROM auction_prices
+    )";
```

Comment:
nit: let's use the common formatting for multi-line queries.
```suggestion
    const QUERY: &str = r#"
SELECT * FROM auction_prices WHERE auction_id = (
    SELECT MAX(auction_id)
    FROM auction_prices
)
    "#;
```